xbmc/pvr/epg/EpgSearchPath.cpp

```diff
@@ -56,6 +57,8 @@ bool CPVREpgSearchPath::Init(const std::string& strPath)
   m_bSavedSearchesRoot =
       (m_bValid && (segments.size() == 4) && (segments.at(3) == "savedsearches"));
   m_bSavedSearch = (m_bValid && (segments.size() == 5));
+  if (m_bSavedSearch)
+    m_iId = std::stoi(segments.at(4));
```

Comment:
Do we need the magic numbers? 

It’s just too much work to fix this is also a fine response 😉

Reply:
Fair, but this is for another PR.